eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -42,6 +42,11 @@ function Check-ApiReviewStatus($packageName, $packageVersion, $language, $url, $
     {
       Write-Host "API Review is approved for package $($packageName)"
     }
+    else if ($response.StatusCode -eq '202')
```

Comment:
Do we need to update any callers of this method to ensure the call for prerelease packages as well?

Reply:
Done